lightning/src/ln/channelmanager.rs

```diff
@@ -6048,6 +6058,8 @@ impl_writeable_tlv_based!(CounterpartyForwardingInfo, {
 	(2, fee_base_msat, required),
 	(4, fee_proportional_millionths, required),
 	(6, cltv_expiry_delta, required),
+	(7, outbound_htlc_minimum_msat, required),
```

Comment:
We shouldnt add new required fields - this will mean we refuse to read objects serialized by older LDK versions.